gits/test/terminal_test.py

```diff
@@ -492,7 +492,7 @@ def _check_cap_cub1(self, pos):
 
         term._cap_cub1()
 
-        if cur_x == 0:
+        if cur_x <= 1:
             self.assertEqual(term._right_most, term._cur_x)
```

Comment:
it's because of:
```python
self._cur_x = max(0, self._cur_x - 1)
if self._cur_x == self._left_most:
            self._cur_x = self._right_most
            // the rest code.
```

there are two cases when the if statement will be true: `cur_x = 0` and `cur_x = 1`.